configure.ac

```diff
@@ -96,6 +96,16 @@ AS_IF([test -n "$missing_libevent"],
       [AC_MSG_ERROR([could not find the libevent libraries])])
 LIBS="$save_LIBS"
 
+# TCMalloc gubbins
+AC_ARG_WITH([tcmalloc],
+            [AS_HELP_STRING([--without-tcmalloc],
+                            [disable tcmalloc for memory allocations])],
+            [],
+            [with_tcmalloc=yes])
+AS_IF([test "x$with_tcmalloc" != xno],
+      [AC_CHECK_LIB([tcmalloc], [malloc],,
+                    [AC_MSG_FAILURE([no tcmalloc found (use --without-tcmalloc to disable)])])])
```

Comment:
It's just because you end up getting a flurry of these knobs, which do something that's specific to the package you're looking at (by which I mean, in this case, "certificate-transparency", not tcmalloc), when these sorts of add-on libraries will usually have instructions on how to enable them with Autotools build systems (with the generic `LDFLAGS`/`LIBS` flags).

These knobs are also kind of hard to do right (which is part of my hatred for them). For example, I think `--without-tcmalloc` won't have any effect, the way you've done it. :wink: 

If you want to be *really* nice, you'd also do them take `--with-tcmalloc=<PATH>`, to get it from that place, which is yet more hassle. So I just escape the whole thing by saying "screw it, if you want a thing in `LIBS`, just put it in `LIBS`".